ee/ucp/admin/configure/route-reflector-deployment.md

```diff
@@ -0,0 +1,210 @@
+# Steps to deploy Route Reflector on UCP 3.0
+
+Note: If you are using UCP 3.0 on Azure deploying route reflectors is not required. The networking control plane is handled
+by Azure SDN and not calico.
+
+1)  On one of the nodes in Docker EE cluster, download the calicoctl binary from 
+    ```
+    curl -O -L https://github.com/projectcalico/calicoctl/releases/download/v3.1.1/calicoctl
+    chmod +x calicoctl
+    $PATH=$PATH:/<directory when calicoctl is downloaded>
+    ```
+    
+2) ```calicoctl``` will need to be configured with the etcd information. There are 2 options:
+
+    a) You can set the environment variables
+    
+       ```
+       export ETCD_ENDPOINTS=127.0.0.1:12379 
+       export ETCD_KEY_FILE=/var/lib/docker/volumes/ucp-node-certs/_data/key.pem 
+       export ETCD_CA_CERT_FILE=/var/lib/docker/volumes/ucp-node-certs/_data/ca.pem 
+       export ETCD_CERT_FILE=/var/lib/docker/volumes/ucp-node-certs/_data/cert.pem 
+       ```
+       
+    b) You can create a configuration file.
+       ```calicoctl``` by default looks at ```/etc/calico/calicoctl.cfg``` for the configuration. For custom 
+       configuration file location, use ```--config``` with ```calicoctl```
+       
+       ```calicoctl.cfg:```
+       ```
+       apiVersion: projectcalico.org/v3
+       kind: CalicoAPIConfig
+       metadata:
+       spec:
+         datastoreType: "etcdv3"
+         etcdEndpoints: "127.0.0.1:12379"
+         etcdKeyFile: "/var/lib/docker/volumes/ucp-node-certs/_data/key.pem"
+         etcdCertFile: "/var/lib/docker/volumes/ucp-node-certs/_data/cert.pem"
+         etcdCACertFile: "/var/lib/docker/volumes/ucp-node-certs/_data/ca.pem"
+       ```
+   
+3) Identify the nodes where the route reflector needs to be deployed. Please note, this node will not be able to schedule
```

Comment:
Please tell me about this right at the start. I might have to provision more nodes, so I need to know about this requirement upfront before I start following this step-by-step instructions.